### ISSUE TITLE
Train longer, generalize better: closing the generalization gap in large batch training of neural network[Hoffer+(Technion),NIPS2017]

### ISSUE THREAD

**Issue by ssytnt** [ssytnt/papers#80]
## 概要
Large-batchで学習した際に汎化性能が低くなるのは、重みの更新回数が少ないことに起因することを実験的に確認。

## 新規性
学習の初期段階をランダムポテンシャル上のランダムウォークでモデル化、‖𝐰_𝑡−𝐰_0 ‖が対数増加すると近似。

## 方法
・学習率のスケーリングを工夫。
・Large-batchを仮想的なSmall-batchに分割(Ghost Batch Normalization)。
・エポックではなく更新回数を統一。

## 結果
メジャーなネットワークとデータセットで分類タスクを実施、バッチサイズと性能の関係を確認。

## 議論
・学習率は‖𝐰_𝑡−𝐰_0 ‖に応じてスケジューリングさせるべき。
・Large-batchで計算時間を短縮することが今後の課題。
・Flatな解ほど中心からの距離が大きいので到達するのに時間を要する(著者の推測)。
・大きなロス値になる局所解が指数関数的に減少すること[Bray+,2007]と関係があると見ている。
![画像2](https://user-images.githubusercontent.com/129141420/229665842-db5846f6-a0d4-4f53-9dec-1088ca0a701a.png)
